data/provider/src/main/java/tech/pegasys/teku/api/ConfigProvider.java

```diff
@@ -92,7 +99,82 @@ public GetSpecResponse getConfig() {
     return new GetSpecResponse(configAttributes);
   }
 
-  private String formatValue(final Object v) {
+  public SerializableTypeDefinition<ConfigProvider> getSpecTypeDefinition() {
```

Comment:
Want to see if I can avoid using the type `ConfigProvider` for the `SerializableTypeDefinition`.